electrumsv/wallet_database.py

```diff
@@ -9,9 +9,10 @@
 is out of date.
 
 """
-
+from __future__ import annotations
```

Comment:
I imported this because otherwise UTXOCache cannot type annotate stuff inside of its own class definition. But it's gone now. Didn't notice until I pushed.